MathLib/LinAlg/Eigen/EigenTools.h

```diff
@@ -11,12 +11,14 @@
 #define EIGENTOOLS_H_
 
 #include <vector>
+#include <Eigen/Sparse> //for RawMatrixType::Index
 
 namespace MathLib
 {
 class EigenMatrix;
 class EigenVector;
 
+using RawMatrixType = Eigen::SparseMatrix<double, Eigen::RowMajor>;
```

Comment:
Isn't that already defined inside the `EigenMatrix` class? I would prefer to reuse that definition.